path_config.go

```diff
@@ -72,6 +73,10 @@ func pathConfig(b *jwtAuthBackend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: "The CA certificate or chain of certificates, in PEM format, to use to validate connections to the JWKS URL. If not set, system certificates are used.",
 			},
+			"jwks_pairs": {
```

Comment:
Like the comment above: it would probably be a more usable config if there is a good way to do a slice of struct here instead of a map. I was having a hard time getting that to parse properly.

Reply:
Ya.. it's a challenge today with Vault backends. There are some ways to make it work but they require us to manually parse outside of the backend framework. Do we anticipate needing any configuration in addition to the JWKS URL and CA?

Reply:
I don't expect any additional config to be added to the pairs.